### PR TITLE
Feature/siiri create user

### PR DESCRIPTION
**Pull request by siiritann**

### CODE REVIEW

File: src/main/java/com/intranet/project/controller/UserController.java [hunk 1]
```diff
@@ -1,4 +1,29 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Map;
+
```
Comment (RohtlaS) [inline src/main/java/com/intranet/project/controller/UserController.java:15]: We should probably already add @CrossOrigin here

File: src/main/java/com/intranet/project/controller/UserController.java [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+@RestController
 public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+
+    // 1 - CREATE ACCOUNT
+    @PostMapping("/createuser")
+    public void createUser(@RequestBody UserCreation userCreation){
```
Comment (ottkaskjr) [inline src/main/java/com/intranet/project/controller/UserController.java:25]: Insted of 'void' we should return an object called ResponseBody(which includes a key named 'message'), which I will create in a moment. Later we should chain this process with login procedure.

File: src/main/java/com/intranet/project/controller/UserController.java [hunk 3]
```diff
@@ -1,4 +1,29 @@
 package com.intranet.project.controller;
```
Comment (ottkaskjr) [inline src/main/java/com/intranet/project/controller/UserController.java:2]: package com.intranet.project.controller.ResponseBody

Comment (ottkaskjr) [inline src/main/java/com/intranet/project/controller/UserController.java:2]: package com.intranet.project.controller.ResponseBody

File: src/main/java/com/intranet/project/repository/UserRepository.java [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+import java.util.Map;
+
+@Repository
 public class UserRepository {
+
+    @Autowired
+    private NamedParameterJdbcTemplate jdbcTemplate;
+
+    // 1 - CREATE ACCOUNT
+    public void createUser(UserEntity userEntity){
```
Comment (ottkaskjr) [inline src/main/java/com/intranet/project/repository/UserRepository.java:19]: should return a String for the ResponseBody.message

File: src/main/java/com/intranet/project/service/UserService.java [hunk 5]
```diff
@@ -1,4 +1,25 @@
...
+
+@Service
 public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+
+    // 1 - CREATE ACCOUNT
+    public void createUser(UserCreation userCreation){
```
Comment (ottkaskjr) [inline src/main/java/com/intranet/project/service/UserService.java:18]: should return a String for the ResponseBody.message

**Review by RohtlaS — CHANGES_REQUESTED** [review]
Check comments in file

File: src/main/java/com/intranet/project/controller/UserController.java (hunk 1 above)
Comment (siiritann) [reply src/main/java/com/intranet/project/controller/UserController.java:15]: done

**Review by ottkaskjr — APPROVED** [review]

**Review by RohtlaS — APPROVED** [review]

**Pull request merged**